src/device-uptime/utils/device_hourly_records.py

```diff
@@ -25,13 +25,17 @@ def __init__(self, tenant, device_name, channel_id):
         self.device_name = device_name
         self.channel_id = channel_id
         self.record = self.get_recent_event()
+        self.token = configuration.AIRQO_API_TOKEN
 
     def get_recent_event(self):
         api_url = f'{configuration.DEVICE_RECENT_EVENTS_URL}?tenant={self.tenant}&channel={self.channel_id}'
 
-        recent_event = requests.get(api_url, verify=False)
+        recent_event = requests.get(api_url, verify=False, headers={
```

Comment:
We are soon (in about one month) adding automatic expiration of JWTs that will require users to login again to generate a new JWT token. How would this token refresh be supported with this implementation? @BenjaminSsempala